subsys/config/src/config_file.c

```diff
@@ -0,0 +1,295 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ * Copyright (c) 2015 Runtime Inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <string.h>
+#include <assert.h>
+#include <zephyr.h>
+
+#include <fs.h>
+
+#include "config/config.h"
+#include "config/config_file.h"
+#include "config_priv.h"
+
+static int conf_file_load(struct conf_store *, load_cb cb, void *cb_arg);
```

Comment:
struct conf_store *cs? other arguments have names, can follow the same with all arguments.

Reply:
fixed